AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/JoinCallActivity.java

```diff
@@ -41,6 +54,13 @@ protected void onCreate(final Bundle savedInstanceState) {
         }
 
         initializeUI();
+
+        mStartForResult = registerForActivityResult(new ActivityResultContracts.StartActivityForResult(),
+            result -> {
+                if (result.getResultCode() == Activity.RESULT_OK) {
+                    finish();
+                }
+            });
```

Comment:
What is this for?

Reply:
once AndroidSettings are opened, we need to go back to InroActivity.